### PR TITLE
added dockerfile, changed channel output

### PR DESCRIPTION
**Pull request by pgbv**

### CODE REVIEW

**Comment by pgbv** [comment]
@HarpyWar check if everything's alright please

File: README.md [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 
-## Installation
+## Docker installation
 
-0. Run `composer install` in the root of the project to download dependencies into `vendor`
+0. Install docker on your system
 
-1. Rename `config.php.example` -> `config.php`
+1. Build the image
+```plaintext
```
Comment (nurupo) [inline README.md]: Perhaps `bash` or `sh` instead of `plaintext`?

**Comment by nurupo** [comment]
Docker and bash changes look good. Not familiar with php, @HarpyWar will have to take a look.

File: update_nfkplanet.php [hunk 2]
```diff
@@ -12,48 +12,73 @@
...
 $p_count = 0;
 // count players
 foreach ($servers as $s)
 {
 	$load = explode('/', $s->load);
 	foreach ($s->players as $p)
 	{
-		if ($p->playerID)
+		// count only confirmed players on tribes servers
+		if (preg_match('/TRX/',$s->name) and $p->playerID)
```
Comment (HarpyWar) [inline update_nfkplanet.php]: `$p_count++;` executes in both cases

**Pull request merged**